shared/AppInsightsCore/src/JavaScriptSDK.Enums/LoggingEnums.ts

```diff
@@ -92,6 +92,7 @@ export const _InternalMessageId = {
     InvalidEvent: 70,
     FailedMonitorAjaxSetRequestHeader: 71,
     SendBrowserInfoOnUserInit: 72,
-    PluginException: 73
+    PluginException: 73,
+    InstrumentationError: 74,
```

Comment:
Where is this used?

Reply:
I appear to have removed the usage I did have in an earlier iteration -- will remove.